Adding two articles from the diversity workgroup. (#215)
pythonindia/inpycon-blog: content/blog/2019-convice-your-boss-with-this-email.md

Title:Convince Your Boss With This Email 
Date:2019-06-01 09:00 
Category:2019 
Slug:2019-convince-your-boss-with-this-email 

> Summary:Save time composing a email to convince your boss to send you to PyCon India. We got you covered.

Reviewer: ```suggestion
Summary:Save time composing an email to convince your boss to send you to PyCon India. We got you covered.
```

> There are a gazillion things to be gained attending PyCon India. Unfortunately, it takes time to sit and compose a neat little email to send to your boss enumerating them. Now consider that atleast half of our attendees attempt to do that, we would lose so many hours of productivity reinventing the wheel ;) . So PyCon India is here to save the day. We have composed a convince-the-boss letter to save you time. Of course, feel free to edit it as you like, remove some, add some or use as is. 

Reviewer: ```suggestion
There are a gazillion things to be gained attending PyCon India. Unfortunately, it takes time to sit and compose a neat little email to send to your boss enumerating them. Now consider that atleast half of our attendees attempt to do that, we would lose so many hours of productivity reinventing the wheel ;) . So PyCon India is here to save the day. We have composed a "Convince The Boss" letter to save you time. Of course, feel free to edit it as you like, remove some, add some or use as is. 
```

## Important Notes
1. You do not need to keep all the points. Remove ones you deem unnecessay. The shorter it is, the better the chances that your boss will read it.

> 2. Watchout for the `{TEMPLATE_VARIABLES}`. You need to replace it with information relevant to you. Don't let one escape you.

Reviewer: ```suggestion
2. Watchout for the `{TEMPLATE_VARIABLES}`. You need to replace them with information relevant to you. Don't let them escape you.
```

* `{OPTIONAL}` As a `{WOMAN/PERSON WITH VISUAL DIFFICULTIES, etc}`, attending the conference will also demonstrate to the tech community that our company takes diversity in our workforce very seriously. Innovative thinking and better products come from teams that are diverse covering more ground.

There is an Early Bird ticket offer on for the first 300 tickets sold before 30th June 2019 and it costs only Rs 1100. Booking later (or after the first 300 tickets) would cost Rs 1600. The tickets include access to all PyCon India keynotes & talks, sponsor showcase, community spaces, an official PyCon India T-shirt along with lunch and high-tea in the evening for both days (12th-13th October).

I would be happy to write a post-conference report detailing what I've learnt and to recommend any changes that we could implement to improve our practices. Please let me know if you have any questions or concerns. I hope we make use this opportunity for the benefit of our company.

Thank you,

`{YOURNAME}`

## Credits

> The article is loosely based on the PyCon India 2018 page

Reviewer: ```suggestion
The article is loosely based on PyCon India 2018's version
```
